src/Mod/AddonManager/addonmanager_workers.py

```diff
@@ -259,11 +259,16 @@ def run(self):
                     except Exception:
                         print("AddonManager: Unable to fetch git updates for repo", repo[0])
                     else:
-                        if "git pull" in gitrepo.status():
-                            self.mark.emit(repo[0])
-                            upds.append(repo[0])
-                            # mark as already installed AND already checked for updates AND update available
-                            self.repos[self.repos.index(repo)][2] = 3
+                        try:
+                            if "git pull" in gitrepo.status():
+                                self.mark.emit(repo[0])
+                                upds.append(repo[0])
+                                # mark as already installed AND already checked for updates AND update available
+                                self.repos[self.repos.index(repo)][2] = 3
+                        except stderr:
+                            FreeCAD.Console.PrintWarning(translate("AddonsInstaller",
+                                                                   "AddonManager - " + repo[0]) + " git status"
```

Comment:
The issue here that may arise is the variable name and it's syntax ` + repo[0]) + ` within the translation string may actually get translated by the translation teams. Also they may not know what `git status` means.